src/DetectChanges/BCBreak/ClassConstantBased/ClassConstantValueChanged.php

```diff
@@ -39,4 +45,20 @@ public function __invoke(ReflectionClassConstant $fromConstant, ReflectionClassC
             true
         ));
     }
+
+    private function getValueWithDirectoryRemoved(ReflectionClassConstant $constant): mixed
+    {
+        $value = $constant->getValue();
+
+        if (! is_string($value)) {
+            return $value;
+        }
+
+        $filePath = $constant->getDeclaringClass()->getFileName();
+        if (! $filePath) {
+            return $value;
+        }
+
+        return str_replace(dirname(realpath($filePath)), self::MAGIC_DIR_OR_FILE_VALUE, $value);
```

Comment:
I'm not 100% convinced this is "the best way":tm: but it was just an idea I put together. Thoughts welcome!

Reply:
Indeed, looks iffy